docs/make.jl

```diff
@@ -0,0 +1,66 @@
+# This file is a part of JuliaFEM.
+# License is MIT: see https://github.com/JuliaFEM/MFrontInterface.jl/blob/master/LICENSE
+
+using Pkg, Documenter, MFrontInterface, Literate, Dates
+
+# automatically generate documentation from tests
+
+"""
+    add_datetime(content)
+
+Add page generation time to the end of the content.
+"""
+function add_datetime(content)
+    line =  "\n# Page generated at " * string(DateTime(now())) * "."
+    content = content * line
+    return content
+end
+
+"""
+    remove_license(content)
+
+Remove licence strings from source file.
+"""
+function remove_license(content)
+    lines = split(content, '\n')
+    function islicense(line)
+        occursin("# This file is a part of JuliaFEM.", line) && return false
+        occursin("# License is MIT:", line) && return false
+        return true
+    end
+    content = join(filter(islicense, lines), '\n')
+    return content
+end
+
+function generate_docs(pkg)
+
+    function preprocess(content)
+        content = add_datetime(content)
+        content = remove_license(content)
+    end
+
+    pkg_dir = dirname(dirname(pathof(pkg)))
+    testdir = joinpath(pkg_dir, "test")
+    outdir = joinpath(pkg_dir, "docs", "src", "tests")
+    test_pages = []
+    for test_file in readdir(testdir)
+        startswith(test_file, "test_") || continue
```

Comment:
`test_` is a problem because you name the folder `test_norton` (or similar). Try yourself by merging the master to this branch.

Reply:
https://travis-ci.com/JuliaFEM/MFrontInterface.jl/jobs/241423656
```
ERROR: LoadError: ArgumentError: cannot find inputfile `/home/travis/build/JuliaFEM/MFrontInterface.jl/test/test_norton_model`
```

Reply:
Hmm yes the problem here is directories.